modules/ibi-install-ibi-operator.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_content-type: PROCEDURE
+[id="ibi-install-ibi-operator_{context}"]
+= Installing the image-based installation Operator
+
+The Image Based Install Operator creates a configuration ISO using site-specific configuration resources that you define in the hub cluster. The IBI Operator boots the target host with the configuration ISO attached to begin the reconfiguration process. 
+
+The IBI Operator also validates the configuration resources that you define in the hub cluster, for example, the Operator checks YAML formatting and verifies networking configurations.
+
+[NOTE]
+====
+The IBI Operator is part of the multicluster engine for Kubernetes operator (MCE) from MCE version 2.10.
+====
+
+.Prerequisites
+
+* You logged in as a user with `cluster-admin` privileges.
+* You deployed a {rh-rhacm-first} hub cluster or you deployed the multicluster engine for Kubernetes operator (MCE).
+* You reviewed the required versions of software components in the section _Software prerequisites for the image-based installation_.
+
+.Procedure
+. Edit the `MultiClusterEngine` resource in the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc edit multiclusterengine
+----
+
+. Set the `enabled` specification to `true` for the `image-based-install-operator-preview` component:
+[source,text]
```

Comment:
This codeblock is one level out compared to the text. Consider adding a `+`

It probably causes the numbers to restart in the next step as it renders as "1. Save the file."